### ISSUE TITLE
mesh

### ISSUE THREAD

**Issue by srem1** [piotr022/UV_K5_playground#12]
Is it possible to add a sound modem, send | receive digital data? on free band like 433 or 868 MHz?

**Comment by fagci** [comment]
There is already support by chip. How you need to create data to transmit? By UART, or keyboard?

**Comment by UT4UUB** [comment]
I think, will be great somethink like KISS TNC over UART. For BBS, Packet Radio, APRS. Is possible?

**Comment by kronicd** [comment]
After testing I have confirmed that BK4819 produces FSK, not AFSK. It will not be useful for general ham radio packet radio (APRS/BBS).

UART FSK may still have some use, but will not be compatible with existing ham infrastructure.

**Comment by bialystok** [comment]
Yes, BBS, Pocket Radio will be great but today is https://reticulum.network/index_pl.html and https://meshtastic.org/ (work on 433MHz) reticullum work on any band.

Big problem is more nodes for any mesh network. 3 keys is fine for typing short text directly with this dev.

**Comment by fab1an2** [comment]
tracker https://hackaday.io/project/183021-lora-aprs-433-mhz-battery-powered-beacon
